LOCALIZATION.md

```diff
@@ -0,0 +1,11 @@
+# Localization
+
+Now that the first version of the cloud native glossary is live, we'd love to start translating it into different languages. Here are the basic requirements for a localization team:
+
+* At least three volunteers
+* Two approvers Each term must be approved by two approvers
+* Unanimous consent must be reached to be approved 
```

Comment:
Minor comment on formatting :)

```suggestion
Now that the first version of the cloud native glossary is live,
we'd love to start translating it into different languages. 

## Basic requirements for a localization team

- At least three volunteers
- At least two approvers 
  - Each term must be approved by two approvers
  - Unanimous consent must be reached to be approved 
```

## Basic requirements for a localization team

- At least three volunteers
- At least two approvers 
  - Each term must be approved by two approvers
  - Unanimous consent must be reached to be approved 